src/drivers/distance_sensor/tfmini/tfmini_main.cpp

```diff
@@ -123,6 +164,14 @@ Attempt to start driver on a specified serial device.
 $ tfmini start -d /dev/ttyS1
 Stop driver
 $ tfmini stop
+Retreive the version on a TFMINI-Plus: (0x5A 0x04 0x01 0x5F)
+$ tfmini command -c 5A04015F
+Configure TFMINI-Plus in Standard 9 bytes (cm) mode: (0x5A 0x05 0x05 0x01 0x65)
+$ tfmini command -c 5A05050165
+Configure TFMINI-Plus Frame Rate: (0x5A 0x06 0x03 0xLL 0xHH 0xSU) - Example for 100Hz
+$ tfmini command -c 5A06036400C7
+Apply and Save Settings for TFMINI-Plus: (0x5A 0x04 0x11 0x6F)
+$ tfmini command -c 5A04116F
```

Comment:
If there are a limited number of commands available, it might be easier for the user to abstract the byte values here by giving them variables/names.

Reply:
> If there are a limited number of commands available, it might be easier for the user to abstract the hex values here by giving them variables/names.

The commands are not limited to any subset. They will be validated for TFmini and TFmini-plus (header and lenght only) and will be rejected if model is not defined.

I will take into account your comments and will push more changes.
Thanks a lot @mcsauder !